src/projections/analysis/SumDetailReader.java

```diff
@@ -41,6 +41,12 @@
     private int numIntervals;
     private int numEPs;
     private long intervalSize;
+	private double[] msg_count; 
+	private double[] msg_size; 
+	private double[] msg_recv_count; 
+	private double[] msg_recv_size; 
+	private double[] msg_recv_count_ext; 
+	private double[] msg_recv_size_ext; 

```

Comment:
These should all be read as RLEBlocks to match the existing way of parsing SumDetail data.